diff -NurbB a/lib/mount_util.c b/lib/mount_util.c
--- a/lib/mount_util.c	2010-03-08 16:18:43.000000000 +0100
+++ b/lib/mount_util.c	2010-05-07 09:52:56.000000000 +0200
@@ -150,9 +150,9 @@
 
 		sigprocmask(SIG_SETMASK, &oldmask, NULL);
 		setuid(geteuid());
-		execl("/bin/mount", "/bin/mount", "--no-canonicalize", "-i",
+		execlp("mount", "mount", "--no-canonicalize", "-i",
 		      "-f", "-t", type, "-o", opts, fsname, mnt, NULL);
-		fprintf(stderr, "%s: failed to execute /bin/mount: %s\n",
+		fprintf(stderr, "%s: failed to execute mount: %s\n",
 			progname, strerror(errno));
 		exit(1);
 	}
@@ -207,9 +207,9 @@
 	if (res == 0) {
 		sigprocmask(SIG_SETMASK, &oldmask, NULL);
 		setuid(geteuid());
-		execl("/bin/umount", "/bin/umount", "-i", rel_mnt,
+		execlp("umount", "umount", "-i", rel_mnt,
 		      lazy ? "-l" : NULL, NULL);
-		fprintf(stderr, "%s: failed to execute /bin/umount: %s\n",
+		fprintf(stderr, "%s: failed to execute umount: %s\n",
 			progname, strerror(errno));
 		exit(1);
 	}
